odoo/addons/base/tests/test_translate.py

```diff
@@ -885,6 +886,7 @@ def test_pattern_conversion(self):
             (r'a\bc',     '%abc%',     r'redundant \ for pattern should be removed'),
             ( 'abc_de',   '%abc%',      'sub patterns less than 3 characters should be dropped'),
             ( 'ab',       '%',          'patterns without trigram should be simplified'),
+            ('Æéç°²ð',    '%Æéç°²ð%',   'Special characters should not be escaped'),
```

Comment:
Might be a good idea to also check very high-range characters e.g. emoji, CJK Compatibility Ideographs Supplement block, ...

Also testing the escaping seems not very interesting as what we're really interested in is *the search*. Maybe that could be tested more extensively with similarly odd search patterns? (with positive and negative matches both)